apps/CA.pl.in

```diff
@@ -7,39 +7,53 @@
 use strict;
 use warnings;
 
-my $openssl = "openssl";
+our ($ME) = ($0 =~ m|/([^/]+)$|);
+our $openssl = "openssl";
 if(defined $ENV{'OPENSSL'}) {
     $openssl = $ENV{'OPENSSL'};
 } else {
     $ENV{'OPENSSL'} = $openssl;
 }
 
-my $verbose = 1;
+our $verbose = 1;
 
-my $OPENSSL_CONFIG = $ENV{"OPENSSL_CONFIG"};
-my $DAYS = "-days 365";
-my $CADAYS = "-days 1095";	# 3 years
-my $REQ = "$openssl req $OPENSSL_CONFIG";
-my $CA = "$openssl ca $OPENSSL_CONFIG";
-my $VERIFY = "$openssl verify";
-my $X509 = "$openssl x509";
-my $PKCS12 = "$openssl pkcs12";
+our $OPENSSL_CONFIG = defined $ENV{'OPENSSL_CONFIG'} ? $ENV{'OPENSSL_CONFIG'} : "";
+our $OPENSSL_CONFIG_PAIRBAG = "-config /etc/pki/tls/openssl-pairbag.cnf";
+our $CERTTOP = defined $ENV{'CERTTOP'} ? $ENV{'CERTTOP'} : "/etc/pki/pairbag";
```

Comment:
I have no idea what the pairbag, but it seems to be something out of scope for OpenSSL...